src/main/scala/strawman/collection/immutable/TreeSet.scala

```diff
@@ -32,14 +42,11 @@ final class TreeSet[A]()(implicit val ordering: Ordering[A])
 
   // From SortedLike
   def range(from: A, until: A): TreeSet[A] = ???
-
-  // From SortedPolyTransforms
-  def map[B](f: (A) => B)(implicit ordering: Ordering[B]): TreeSet[B] = ???
```

Comment:
It's not though. The error message is misleading. I haven't tried to track this down further but there is no abstract `map` method anywhere in the inheritance hierarchy of `TreeSet`. It inherits two concrete implementations but without enforcing `CIPS <: IPS` the types don't line up.